artman/cli/main.py

```diff
@@ -113,6 +111,14 @@ def parse_args(*args):
              '/path/to/config.yaml:config_section_A|config_section_B. '
              'This is an advanced feature; for most cases, use `--api`.',
     )
+    api.add_argument('--batch',
+        action='store_true',
+        help='Enables batch mode, which will generate all APIs for the '
```

Comment:
Before we had the batch support to generate all languages for one API. Is this functionality still available? I feel it is somewhat useful to refresh the API for all languages (e.g. proto update) with one single command.